.tav.yml

```
@@ -254,3 +254,7 @@ fastify:
 finalhandler:
   versions: '*'
   commands: node test/instrumentation/modules/finalhandler.js
+
+memcached:
+  versions: '>2.2.0'
```

I just did a quick test and it looks to me like we support all the way down to v2.2.0, so this should be changed to:

```suggestion
  versions: '>=2.2.0'
```

(which is also what the if-statement in the patch file says)